rpc/api.proto

```diff
@@ -64,6 +64,23 @@ service SeedService {
 	rpc DecodeSeed (DecodeSeedRequest) returns (DecodeSeedResponse);
 }
 
+service TicketBuyerService {
+    rpc StartTicketPurchase (StartTicketPurchaseRequest) returns (StartTicketPurchaseResponse);
+    rpc StopTicketPurchase (StopTicketPurchaseRequest) returns (StopTicketPurchaseResponse);
+
+    rpc Config (TicketBuyerConfigRequest) returns (TicketBuyerConfigResponse);
```

Comment:
the request and response message type names should be the name of the method followed by Request or Response, like all of the other ones.  I would change the method name in this case (to TicketBuyerConfig).